src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
```

Comment:
A few comments about this:
1. This element belongs to the navigation block, so this rule should not be nested inside the __items_list rule, but rather go after it

2. While it's a possible solution to set the distance between elements using margin, I'd recommend avoiding this as much as possible and using the possibilities that flex and grid displays introduce (gaps)

Not only the parent will control the display, but also, we'll also avoid setting unnecessary rules, for the first and last elements (see lines 28-34)